### ISSUE TITLE
[Adjust Rule] SIM115 false positive in Django FileResponse

### ISSUE THREAD

**Issue by putianyi889** [MartinThoma/flake8-simplify#197]
## Desired change

* **Rule(s)**: SIM115
* **Adjustment**: Syntax like `FileResponse(open("myfile.png", "rb"))` should be ignored.

## Explanation

By [Django documentation](https://docs.djangoproject.com/en/5.1/ref/request-response/#fileresponse-objects), it is the recommended usage of FileResponse, as
> The file will be closed automatically, so don’t open it with a context manager.

## Example

This is an example where the mentioned rule(s) would currently be suboptimal:

```python
from django.http import FileResponse
response = FileResponse(open("myfile.png", "rb"))
```
